std_msgs/Header header
float64 ctl_force_low              # ctl_force_low reading
float64 ctl_force_low_variance     # 0 is interpreted as unknown

